src/integrationtests/java/com/ericsson/ei/integrationtests/notification/FailedNotificationStepsIT.java

```diff
@@ -155,4 +179,18 @@ protected int extraEventsCount() {
         eventNames.add("event_EiffelTestCaseTriggeredEvent_3");
         return eventNames;
     }
+
+    private void waitForDatabaseEntry(String subscriptionName) throws InterruptedException {
+        final MongoCondition condition = MongoCondition.subscriptionNameCondition(subscriptionName);
+
+        long stopTime = System.currentTimeMillis() + SECONDS_30;
+        ArrayList<String> find = new ArrayList<>(0);
+        do {
+            TimeUnit.MILLISECONDS.sleep(1000);
+            find = mongoDBHandler.find("FailedNotification", "missed_notification_failed_notification", condition);
```

Comment:
Should these be constants instead to make them easier to find? Also I would prefer if we rename the collection name completely to failed_notifications since that is the refactored name we should be using.

Reply:
Fixed in 93d48c9